js/views/contentScreen.js

```diff
@@ -45,12 +46,12 @@ var ContentScreen = React.createClass({
 
     return (
       <div className={this.props.screenClassName}>
+        <Watermark {...this.props} controlBarVisible={false} nonClickable={true}/>
```

Comment:
@mistermendez, the watermark is not clickable from CC, share, bitrate selection screens, but we still show it.